sys/vane/jael.hoon

```diff
@@ -1179,6 +1478,41 @@
     ++  file-dns
       |=  dns=dnses
       ..file(dns.eth dns)
+    ::
+    ++  file-snap                                       ::  save snapshot
+      |=  wer=event-id
+      ^+  ..file
+      =?    sap
+          %+  lth  2
+          %+  sub.add
+            (div block.wer interval.sap)
+          (div last-block.sap interval.sap)
+        ~&  :*  %snap  count=count.sap  max-count=max-count.sap 
+                last-block=last-block.sap  interval=interval.sap
+                lent=(lent ~(tap to snaps.sap))
```

Comment:
Can we just list these purely vertically, for readability's sake? Thanks!

Reply:
Will do